x-pack/test/functional/page_objects/upgrade_assistant_page.ts

```diff
@@ -61,22 +58,20 @@ export function UpgradeAssistantPageProvider({ getPageObjects, getService }: Ftr
       });
     }
 
-    async expectIssueSummary(summary: string) {
-      return await retry.try(async () => {
-        log.debug('expectIssueSummary()');
-        const summaryElText = await testSubjects.getVisibleText('upgradeAssistantIssueSummary');
-        expect(summaryElText).to.eql(summary);
+    async waitForTelemetryHidden() {
+      const self = this;
+      retry.waitFor('Telemetry to disappear.', async () => {
```

Comment:
I'm seeing failures like https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/104171/execution/node/796/log/ 

What seems to happen is that tests pass and complete, so the test runner kills the kibana process. However webdriver is still stuck in this waitFor and so exits with a non-zero code.

I suspect we need to insert an await here `await retry.waitFor('...` so that we don't end the test and kill Kibana before webdriver is done.

Reply:
Also needed here: https://github.com/elastic/kibana/pull/89942/files#diff-76765ff6f781aafb2b02c06ab2841e446b19b748b99ac7103aa67f0c0a718da5R26

Reply:
https://github.com/elastic/kibana/pull/90456